jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelListeners.java

```diff
@@ -45,6 +45,7 @@ public class HttpChannelListeners implements HttpChannel.Listener
 
     private static final MethodType LISTENER_TYPE_ON_RESPONSE_COMMITTED = methodType(Void.TYPE, Request.class, Integer.TYPE, HttpFields.class);
     private static final MethodType LISTENER_TYPE_ON_RESPONSE_WRITE = methodType(Void.TYPE, Request.class, Boolean.TYPE, ByteBuffer.class);
+    public static final MethodHandle EMPTY_ON_RESPONSE_WRITE_HANDLE = MethodHandles.empty(LISTENER_TYPE_ON_RESPONSE_WRITE);
```

Comment:
Don't do this, you risk pinning a classloader using `MethodHandles.empty()` as a static final.

Reply:
@joakime Huh? This class already has plenty of static final `MethodHandle`s that are on actual classes?  How can an empty handle pin a classloader?

Reply:
@joakime I changed to use a boolean anyway.